### ISSUE TITLE
Nested url params are broken

### ISSUE THREAD

**Issue by vit-panchuk** [johnsonj/progressive_render#23]
Your gem uses `URI.encode_www_form` to encode `request.GET` in `ProgressiveRender::Rack::RequestHandler#load_path` that does not afford to pass nested hashed like this - `{'grid'=>{'page'=>2}`.
The better solution is to use `::Rack::Utils.build_nested_query` instead.

Eg:
```
# request.GET == {'grid'=>{'page'=>2}
URI.encode_www_form request.GET # => "grid=%7B%22page%22%3D%3E%222%22%7D"
Rack::Utils.build_nested_query request.GET # => "grid[page]=2"
```

**Comment by johnsonj** [comment]
Fix released in `0.7.0`

**Comment by vit-panchuk** [comment]
Cool, thank you!
